pyomo/common/getGSL.py

```diff
@@ -18,7 +18,7 @@
 logger = logging.getLogger('pyomo.common')
 
 # These URLs were retrieved from
-#     https://ampl.com/resources/extended-function-library/
+#     https://ampl.com/resources/extended-function-library/ 
```

Comment:
Not a real change, I recommend checking out this file from master

Reply:
Ah yes - the changes made in Master to remove the "www" in the download lines I had done on my local copy, I guess with a tiny space added. Has been fixed.